src/runtime/threading_backend.cc

```diff
@@ -220,7 +235,7 @@ class ThreadGroup::Impl {
             } else {
               core_id = sorted_order_[i + exclude_worker0];
             }
-            SetThreadAffinity(threads_[i].native_handle(), {core_id});
+            SetThreadAffinity(threads_[i].native_handle(), threads_tid_[i], {core_id});
```

Comment:
This part seems like have a timing issue, threads_tid_ share by main thread and worker threads, current logic not guarantee that the threads_tid_[i] is valid when main thread to read it. I think we should can reproduce such issue by adding a debug "sleep" between line 118 and line 119